src/main/java/org/broadinstitute/hellbender/tools/spark/utils/LongBloomFilter.java

```diff
@@ -107,6 +110,10 @@ public static long getOptimalNumberOfBits(final long numElements, final double f
         return (long) Math.ceil(-numElements * Math.log(fpp) / (Math.log(2) * Math.log(2)));
     }
 
+    public double getTheoreticalFPP(final long numElements) {
+        return Math.pow(1.0 - Math.pow(1.0 - (1.0/totalBits), numHashes * numElements), numHashes);
+    }
+
```

Comment:
A few lines below here, on line 125 you have:
```sum += bitCountMap[0 & b];```
That's clearly wrong -- 0 & b is always 0.